### PR TITLE
services/horizon/internal/actions: add extra test case to TestGetPageQueryCursorDefaultTOID

### PR DESCRIPTION
**Pull request by tamirms**
<details>
  <summary>PR Checklist</summary>
  
### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
* [ ] I've updated any docs ([developer docs](https://developers.stellar.org/api/), `.md`
  files, etc... affected by this change). Take a look in the `docs` folder for a given service,
  like [this one](https://github.com/stellar/go/tree/master/services/horizon/internal/docs).

### Release planning

* [ ] I've updated the relevant CHANGELOG ([here](services/horizon/CHANGELOG.md) for Horizon) if
  needed with deprecations, added features, breaking changes, and DB schema changes.
* [ ] I've decided if this PR requires a new major/minor version according to
  [semver](https://semver.org/), or if it's mainly a patch change. The PR is targeted at the next
  release branch if it's not a patch change.
</details>

### What

Add extra test case to TestGetPageQueryCursorDefaultTOID which ensures that we don't set the cursor to a negative ledger sequence when the oldest ledger in the db is 0.

### CODE REVIEW

File: services/horizon/internal/actions/helpers_test.go [hunk 1]
```diff
@@ -346,6 +346,25 @@ func TestGetPageQueryCursorDefaultTOID(t *testing.T) {
 	assert.Equal(t, uint64(2), pq.Limit)
 	assert.Equal(t, "desc", pq.Order)
 
+	ledgerState.SetHorizonStatus(ledger.HorizonStatus{
```
Comment (urvisavla) [inline services/horizon/internal/actions/helpers_test.go:349]: Isn't this already covered in [330-347](https://github.com/stellar/go/pull/5411/files#diff-80eb3baad7bd4c02643f313b311fad45de2de20e1fcb4af0b4e50e16bb53243fR330-R347)?

Comment (tamirms) [reply services/horizon/internal/actions/helpers_test.go:349]: 🤦 yes, you're right. I'll close the PR!
